app/src/main/java/com/teamwss/websoso/ui/onBoarding/model/OnBoardingPage.kt

```diff
@@ -0,0 +1,15 @@
+package com.teamwss.websoso.ui.onBoarding.model
+
+enum class OnBoardingPage(val progressPercent: Int, val isBackButtonVisible: Boolean, val isSkipTextVisible: Boolean) {
```

Comment:
아하 제가 생각했던 건
```suggestion
class Page {
    private val _page = OnboardingPage.FIRST
    val page get() = _page  // 실시간 감지가 필요하다면 라이브데이터
    
    fun updateNextPage() {
         _page.value = ....
    }
    
    fun updatePreviousPage() {
         _page.value = ....
    }
}
```

외부에선 다음페이지, 이전페이지 업데이트 함수만 호출하고, 실제 업데이트는 해당 객체내부에서 캡슐화되어 수행하게끔
페이지에 대한 상태는 외부에 노출..
관련 로직은 해당 객체 내부에 캡슐화하는 식으로 생각해보았습니다.